components/image-builder-mk3/pkg/orchestrator/orchestrator.go

```diff
@@ -150,7 +151,7 @@ func (o *Orchestrator) ResolveBaseImage(ctx context.Context, req *protocol.Resol
 	tracing.LogRequestSafe(span, req)
 
 	reqs, _ := protojson.Marshal(req)
```

Comment:
Handler errors?

Reply:
I think all debug logs in this file can be deleted, since they are going to log `[redacted:nested]` anyway. Alternatively another data structure should be used implementing `Scrubbable` then we don't need to fiddle with marshal/unmarshall here. cc @kylos101 not sure what is the best here

Reply:
actually looking at the code it used to be the same before, so it seems errrs are skipped there on purpose

Reply:
Will `scrubber.Default.JSON` not panic when it receives `nil` as `reqs`?

Reply:
maybe, it seems it never happened 🤷‍♂️ new and old code are identical, they both will do

```
var v any
err := json.Unmarshal(nil, v)
```

json.Unmarshal return an err in this case but does not panic

i will leave it to the workspace team to decide, I think the semantic here as before.

Reply:
I've deleted them because they are scrubbed completely anyway.